### PR TITLE
Fix medical storage whitelists

### PR DESCRIPTION
**Pull request by tsp01**
## About the PR
This PR allows both the storage of blood in several medical pouches and belts, as well as allowing the storage of surgical cases in the medical kit pouch. 
Fixes: #2514

## Why / Balance
these things are meant to fit based on CM13

## Technical details
Prior, the only way to fit blood bags was by listing the BloodPack component, which for several items was mistakenly put as a tag in the whitelists. I have added the tag CMBloodPack to iv.yml, which can now be used to whitelist bloodpacks as opposed to using the component. I have created the tag CMSurgicalCase which can be used to whitelist  surgical cases.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl:
- fix: blood can be stored in the correct pouches and belts
- fix: surgical cases can be stored in the correct pouches and belts

### CODE REVIEW

File: Content.Shared/Storage/EntitySystems/SharedStorageSystem.cs [hunk 1]
```diff
@@ -917,7 +917,8 @@ public void UpdateAppearance(Entity<StorageComponent?, AppearanceComponent?> ent
 
         if (TryComp<StorageComponent>(insertEnt, out var insertStorage)
             && GetMaxItemSize((insertEnt, insertStorage)) >= maxSize
-            && !CMStorage.IgnoreItemSize((uid, storageComp), insertEnt))
+            && !CMStorage.IgnoreItemSize((uid, storageComp), insertEnt)
+            && !_whitelistSystem.IsWhitelistPass(storageComp.Whitelist, insertEnt))
```
Comment (DrSmugleaf) [inline Content.Shared/Storage/EntitySystems/SharedStorageSystem.cs:921]: You don't need to change this, use the IgnoreContentsSize component and then set its items whitelist to surgical cases.

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

**Comment by JoeHammad1844** [comment]
can you store blood packs in armor? i remember being able to do that in CM13

File: Content.Shared/Storage/EntitySystems/SharedStorageSystem.cs (hunk 1 above)
Comment (tsp01) [reply Content.Shared/Storage/EntitySystems/SharedStorageSystem.cs]: I see. Thank you for letting me know. Changed.

**Comment by tsp01** [comment]
> can you store blood packs in armor? i remember being able to do that in CM13

Doesn't seem like it from my quick look at CM13 code. Though that can be added if maints want.

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

**Pull request merged**